libvast/src/system/spawn_metastore.cpp

```diff
@@ -46,4 +48,19 @@ maybe_actor spawn_metastore(caf::local_actor* self, spawn_arguments& args) {
   return caf::actor_cast<caf::actor>(s);
 }
 
+maybe_actor spawn_metastore_simple(caf::local_actor* self, spawn_arguments& args) {
+  auto store = self->spawn(simple_store, args.dir / "simple_store");
```

Comment:
Why is the directory name `simple_store` and not just `store`? I think the implementation detail should not shine through at the filesystem level.

Reply:
The directory structure used for persisting is not part of the interface so I don't know what to do with this objection.
Besides, the name `store` is already occupied by the interface for the data persistence back end in the ARCHIVE. I don't think there is ever a justification for a name clash like this. The name of this directory can be chosen based on the purpose of the instance. My proposal would be `importer` or `importer_state`.
We should also come up with a better name for the type. But that's out of scope of this PR.